receiver/k8sclusterreceiver/internal/node/nodes.go

```diff
@@ -46,23 +46,30 @@ func Transform(node *corev1.Node) *corev1.Node {
 func GetMetrics(set receiver.CreateSettings, node *corev1.Node, nodeConditionTypesToReport, allocatableTypesToReport []string) pmetric.Metrics {
 	mb := imetadata.NewMetricsBuilder(imetadata.DefaultMetricsBuilderConfig(), set)
 	ts := pcommon.NewTimestampFromTime(time.Now())
+	customScopeMetrics := pmetric.NewMetrics().ResourceMetrics().AppendEmpty().ScopeMetrics().AppendEmpty()
```

Comment:
nit:
```suggestion
	customMetrics := pmetric.NewMetricSlice()
```
Then you can use `customMetrics` instead of `customScopeMetrics.Metrics()`

Reply:
Thanks, fixed!